infrastructure/templates/public-api/components/network.bicep

```diff
@@ -9,141 +9,29 @@ param location string
 
 //Specific parameters for the resources
 @description('Virtual Network Address Prefix')
-param vnetAddressPrefix string = '10.0.0.0/16'
-
-@description('Admin Subnet Address Prefix')
-param adminSubnetPrefix string = '10.0.0.0/24'
-
-@description('Importer Subnet Address Prefix')
-param importerSubnetPrefix string = '10.0.1.0/24'
-
-@description('Publisher Subnet Address Prefix')
-param publisherSubnetPrefix string = '10.0.2.0/24'
-
-@description('Content Subnet Address Prefix')
-param contentSubnetPrefix string = '10.0.4.0/24'
-
-@description('DataBase Subnet Address Prefix')
-param databaseSubnetPrefix string = '10.0.5.0/24'
-
-@description('Deploy the subnets')
-param deploySubnets bool = true
+param vNetAddressPrefix string = '10.0.0.0/16'
 
 //Passed in Tags
 param tagValues object
 
 // Variables and created data
 var vNetName = '${subscription}-vnet-${environment}'
-var adminSubnetName = '${subscription}-snet-${environment}-admin'
-var importerSubnetName = '${subscription}-snet-${environment}-importer'
-var publisherSubnetName = '${subscription}-snet-${environment}-publisher'
-var contentSubnetName = '${subscription}-snet-${environment}-content'
-var databaseSubnetName = '${subscription}-snet-${environment}-database'
 
 
 //Resources 
-resource virtualnetwork 'Microsoft.Network/virtualNetworks@2023-04-01' = if (deploySubnets) {
+resource vNet 'Microsoft.Network/virtualNetworks@2023-04-01' = {
```

Comment:
Yeah, I've left any that we're not using in place for now - I figured that since Paul's done the work to modify the off-the-shelf module files to take into account things like our resource prefixes etc, I'd leave them in case we need them in the future.

Others that we're not using include:
- blobStore.bicep
- containerRegistry.bicep
- keyVault.bicep
- serviceBusQueue.bicep
- storageAccount.bicep

Not too fussed though if we decide to remove them for now given that we're not using them - I'm easy!

Reply:
Cool. Yep it's no problem to leave these in for now